vsdkx/addon/zoning/processor.py

```diff
@@ -170,7 +170,7 @@ def post_process(self, addon_object: AddonObject) -> AddonObject:
         zone_count[rest_zone_str] = rest_zone_dict
         inference.extra["zoning"] = zone_count
         addon_object.inference = inference
-
+        print(f'Zoning processor results {inference.extra["zoning"]}')
```

Comment:
Logging should be done by logging package of python so that we can have more control on the level of logs.

Reply:
we have some bug there, and logging is not working for unknown reason :)

Reply:
@a-ghias-natix I actually meant to remove this line, I only added it for debug purposes for this PR

Reply:
Ok, then let's remove this line or fix it by using the logging package and @gujjs let's talk on why logging is not working cause it shouldn't have any problem.